gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -163,6 +165,14 @@ def main(args):
         ht = vep_or_lookup_vep(ht, vep_version=args.vep_version)
         ht.write(res.vep_ht.path, overwrite=args.overwrite)
 
+    if args.valid_vep:
+        vep_ht = hl.read_table(
+            resources.run_vep.vep_ht.path
+        )  # TODO: can I use the vep_ht from the previous step?
+        logger.info("importing and parsing interval file: ")
```

Comment:
I don't think this logger is needed since it's really just using the resource and is no longer imported and parsed at this point
```suggestion
```